### PR TITLE
Added more Syndicate Uniforms to the Emagged Uniform Printer

### PR DESCRIPTION
**Pull request by Smugman**
## About the PR
First PR Time.

Ive Added the Delta-V syndicate uniforms to the Emagged uniform printer.
Specifically:  the Cybersun Lawyer Jumpsuit, the Interdyne Jumpsuit, Chemsuit and Gas mask, and the Cybersun Surgeon Jumpsuit, Surgical Cap and Windbreaker. 

Ive also already added the recipes for [PR#2285](https://github.com/DeltaV-Station/Delta-v/pull/2285)'s Additional items, but have commented them out until it is merged, to be uncommented after testing that they actually work

## Why / Balance
Why? The Upstream Uniform printer lets you print every syndicate uniform they have, Parity with DeltaVs new ones makes sense.

Now, This does allow people to print the Windbreaker, which has a 5% dmg resistance to pierce, slash, blunt and 20% Heat
and the Chemsuit which has 60% Caustic Armor with a very Civilian machine, I personally think that there are far more powerful things one can do with an EMAG and that these vanity items pale in comparrison.

In addition, this PR also allows the uniform printer to create the Interdyne gasmask, which is not only a bit unfitting for a uniform printer, but also currently the only method i know of to make gas masks. 

I did not wish to split up the uniforms into different machines or leave them incomplete, so i think this is fine considering the Price of Admission (an EMAG, plus stealing or Making a Uniprinter)

## Technical details
YML  Changes only, modified two Upstream files, but i believe ive commented them appropriately, recipes for items that dont exist yet commented out.

## Media
image

## How to Test
- [ ] Emag a Uniform Printer
- [ ] Print Some Valids 

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
:cl:
- add: Added more recipes to the Emagged Uniform Printer, print those valids!

### CODE REVIEW

**Comment by Lyndomen** [comment]
Howdy! We are taking this under for direction review, expect a response in 24 hours. Looks good at first glance!

**Review by Lyndomen — DISMISSED** [review]
Direction approved!

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -1269,6 +1269,18 @@
...
+      - ClothingUniformCybersunAttorney #Delta-V Cybersun Lawyer Jumpsuit
+      - ClothingHeadHatSurgcapCybersun #Delta-V Cybersun Surgeon Cap
+      - ClothingOuterInterdyneChemistrySuit #Delta-V Interdyne Outerwear
+      - ClothingOuterCoatCybersunWindbreaker #Delta-V Cybersun Surgeon Outerwear
+      - ClothingMaskInterdyneChemistry #Delta-V Interdyne Gasmask
+      - ClothingHeadHatSurgcapCybersun #Delta-V Cybersun Surgeon Cap
+      #- ClothingUniformCybersunHazard #Delta-V Cybersun Engineering Jumpsuit
+      #- ClothingUniformCybersunCasual #Delta-V Cybersun Casual Jumpsuit
+      #- ClothingUniformCybersunRND #Delta-V Cybersun Researcher Jumpsuit
+      #- ClothingOuterCybersunOvercoat #Delta-V Cybersun Lawyer Overcoat
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml]: just do  a Begin DeltaV Additions and End DeltaV Additions comment at the start and end, not on every individual line

File: Resources/Prototypes/Recipes/Lathes/clothing.yml [hunk 2]
```diff
@@ -570,6 +570,38 @@
   id: ClothingUniformJumpsuitPyjamaSyndicateRed
   result: ClothingUniformJumpsuitPyjamaSyndicateRed
 
+#Start DeltaV
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Lathes/clothing.yml]: move all these recipes to the deltav folder

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml (hunk 1 above)
Comment (Smugman) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml]: this is how i saw it done with the CJ outfits, Ill do it with Start and End now instead.

File: Resources/Prototypes/Recipes/Lathes/clothing.yml (hunk 2 above)
Comment (Smugman) [reply Resources/Prototypes/Recipes/Lathes/clothing.yml]: Moved.

**Review by deltanedas — APPROVED** [review]

**Comment by JustAnOrng** [comment]
Howdy, as much as I appreciate this, I don't appreciate using the art without asking me (Or crediting me, which is fine (but still kinda annoying) since the original PR is linked). I'm totally fine with it, and I'm sure you didn't have any ill intentions.

Unless this PR isn't adding the clothes or something and I'm stupid. Also, don't forget to add the copyright stuff when adding the clothes, and I'm the one who made em! @JustAnOrange on Discord if you have more questions about it, i'm not really that deep into SS14 atm.

**Comment by JustAnOrng** [comment]
nevermind im a stupid moron and this isn't adding the clothes at all and i am dumb
rip

**Pull request merged**